config-linux.md

```diff
@@ -300,7 +300,12 @@ For more information, see the kernel cgroups documentation about [cpusets][cgrou
 The following parameters can be specified to set up the controller:
 
 * **`shares`** *(uint64, OPTIONAL)* - specifies a relative share of CPU time available to the tasks in a cgroup
-* **`quota`** *(int64, OPTIONAL)* - specifies the total amount of time in microseconds for which all tasks in a cgroup can run during one period (as defined by **`period`** below)
+* **`quota`** *(int64, OPTIONAL)* - specifies the replenished amount of time in microseconds for which all tasks in a cgroup can run during one period (as defined by **`period`** below)
+    A value of `-1` indicates that the cgroup does not have any bandwidth restriction in place (i.e., an unconstrained bandwidth group).
+    Any (valid) positive value(s) no smaller than `burst` will enact the specified bandwidth limit.
+* **`burst`** *(uint64, OPTIONAL)* - specifies the maximum amount of accumulated time in microseconds for which all tasks in a cgroup can run additionally for burst during one period (as defined by **`period`** below)
+    A value of `0` for indicates that the cgroup can not accumulate any unused bandwidth. It makes the traditional bandwidth control behavior unchanged.
+    Any (valid) positive value(s) no larger than `quota` will enact the cap on unused bandwidth accumulation.
```

Comment:
Just a question on this line (actually the original linux commit (https://github.com/torvalds/linux/commit/f418371#diff-cc1a82129952a910fdc4292448c2a097a2ba538bebefcf3c06381e45639ae73eR9814), why values larger than quota should return EINVAL? Do you know the reason?

Reply:
@m-yosefpor Thanks for the question!

In short, a burst value much larger than the quota is breaking the design purpose of bwc (as isolation). The limit put on the burst aims to control the interference to other groups.

FYI, this was discussed in the v4 patch set of this feature: https://lwn.net/ml/linux-kernel/YFG4hEOe65cbCo26@hirez.programming.kicks-ass.net/.

Reply:
@kailun-qin Thanks for your explanation. Completely makes sense.